code/processes/dqe.q

```diff
@@ -17,23 +17,21 @@ runcheck:{[id;fn;vars;rs]
   if [not fncheck[2] in key value .Q.dd[`;fncheck 1];                                                           /- run check to make sure passed in function exists
     .lg.e[`function;"Function ",(string fn)," doesn't exist"];
     :()];
-  
+
   rs:(),rs;                                                                                                     /- set rs to a list
   h:.dqe.gethandles[rs];                                                                                        /- check if processes exist and are valid
-  
-  {[h;rs]                                                                                                       /- fill procname for results table
+
+  r:raze{[h;rs]                                                                                                 /- fill procname for results table
```

Comment:
Is it possible to extract out some of these functions separately?

Right now it has a function within a function within a function?  I'd be keen to make them individual if possible?